examples/echo.rs

```diff
@@ -2,18 +2,18 @@
 //!
 //! Very similar to the example at https://tokio.rs
 
-#![feature(proc_macro, proc_macro_non_items, generators)]
+#![feature(generators, proc_macro_hygiene)]
 
 extern crate futures_await as futures;
 extern crate tokio_core;
 extern crate tokio_io;
 
 use std::io::{self, BufReader};
 
-use futures::prelude::*;
+use futures::prelude::{r#await, *};
```

Comment:
If you activate the `await_macro` feature `await` will not be a keyword (at least at the moment).

Reply:
I think adapting to "standard" language syntax rather than bringing in more features is the better way forward.